google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -244,4 +254,16 @@ public void testLifecycleRules() {
     assertTrue(setStorageClassLifecycleRule.getCondition().getIsLive());
     assertEquals(10, setStorageClassLifecycleRule.getCondition().getNumNewerVersions().intValue());
   }
+
+  @Test
+  public void testIamConfiguration() {
+    Bucket.IamConfiguration iamConfiguration = BucketInfo.IamConfiguration.newBuilder()
+            .setIsBucketPolicyOnlyEnabled(true)
+            .setLockedTime(System.currentTimeMillis())
+            .build()
+            .toPb();
+
+    assertEquals(Boolean.TRUE, iamConfiguration.getBucketPolicyOnly().getEnabled());
+    assertNotNull(iamConfiguration.getBucketPolicyOnly().getLockedTime());
```

Comment:
This is still using the old way I think?

Reply:
It's not using the old way, it's just that toPB() is called here so it's using the generated model class which has the classes set up that way

Reply:
acking. missed the Apiary class path. Thanks for clarifying!